DFC.Composite.Shell.Models/PostResponseModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace DFC.Composite.Shell.Models
+{
+    public class PostResponseModel
+    {
+        public string HTML { get; set; }
```

Comment:
Make this property Pascal case or Sonar Cloud will issue a naming convention warning